src/main/java/org/mule/service/http/impl/service/server/ErrorRequestHandler.java

```diff
@@ -37,7 +38,12 @@ public ErrorRequestHandler(int statusCode, String reasonPhrase, String entityFor
 
   @Override
   public void handleRequest(HttpRequestContext requestContext, HttpResponseReadyCallback responseCallback) {
-    String resolvedEntity = getResolvedEntity(requestContext.getRequest().getUri());
+    String resolvedEntity;
+    try {
+      resolvedEntity = getResolvedEntity(requestContext.getRequest().getUri().toString());
+    } catch (IllegalArgumentException e) {
```

Comment:
why is this needed?

Reply:
The change in tests suggests `getPath()` should be used here. We don't actually want to expose anything else.

Reply:
In case of invalid URI, trying to resolve it will result in an `IllegalArgumentException`, making the ErrorRequestHandler not handle the exception